Packs/CyberArkPAS/Integrations/CyberArkPAS/CyberArkPAS.yml

```diff
@@ -0,0 +1,913 @@
+category: Authentication
+commonfields:
+  id: CyberArkPAS
+  version: -1
+configuration:
+- display: Server URL (e.g. https://example.net)
+  name: url
+  required: true
+  type: 0
+- display: Username
+  name: credentials
+  required: true
+  type: 9
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- defaultvalue: '50'
+  display: Max fetch
+  name: max_fetch
+  required: false
+  type: 0
+- defaultvalue: 7 days
+  display: First fetch timestamp (<number> <time unit>, e.g., 12 hours, 7 days, 3
+    months, 1 year)
+  name: fetch_time
+  required: false
+  type: 0
+- defaultvalue: 0
+  display: Query, CyberArk Pas value (0.0-100.0)
+  name: query
+  required: false
+  type: 0
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: This is the Hello World integration for getting started.
```

Comment:
copy pasta

Reply:
please re think the `This is the....`. if you feel you need help, @Arsenikr or maybe alex will have better suggestions.